### DISCUSSION TITLE
Adding Usage stats in users dashboard

### DISCUSSION THREAD

**Discussion by ips972** [open-webui/open-webui#8207] [Ideas]
I believe it would be beneficial to include a chat/usage counter as a column before the edit/delete options on the right in the users and groups dashboard. This would provide valuable statistics on peak and low user engagement within the system.

**Comment by ips972** [comment]
also, one more idea,  in the users list, can we add a "connected" green icon and be able to filter of connected users?  since in the admin menu, the system does show amount of connected users, why not take it a few more steps forward?

**Comment by fkoehne** [comment]
While I share your curiosity, this would make it more difficult to introduce the tool in companies in Europe (feels like a surveillance feature / Betriebsrat does not approve).

**Reply by Classic298** [reply]
on Teams you also see who is online etc.
and on any other messenger

**Reply by jeannotdamoiseaux** [reply]
Ideally, statistics should be presented in an aggregated form and not be linked to individual users. This approach helps to protect user privacy.
